frb_codegen/src/generator/dart/ty_boxed.rs

```diff
@@ -63,7 +62,7 @@ impl TypeDartGeneratorTrait for TypeBoxedGenerator<'_> {
             | RustOpaque(_)
             | EnumRef(_)
             | Primitive(IrTypePrimitive::I64 | IrTypePrimitive::U64 | IrTypePrimitive::Usize)
-            | Delegate(IrTypeDelegate::Array(_)) => {
+            | Delegate(IrTypeDelegate::Array(_) | IrTypeDelegate::PrimitiveEnum { .. }) => {
```

Comment:
nit (unrelated to this line of code): so, have our tests covered *all* cases that this PR changes?